omrmakefiles/rules.ztpf.mk

```diff
@@ -42,7 +42,7 @@ endif
 ### Global Flags
 ###
```

Comment:
The copyright needs to be changed in this file too.

Reply:
Done.